src/store.js

```diff
@@ -42,21 +44,15 @@ export default () => {
             getParticipantById: (state) => (id) => {
                 let curr_participant;
                 state.participants.forEach(participant => {
-                    if (participant.id == id) {
+                    if (participant.id === id) {
                         curr_participant = participant;
                     }
                 });
 
                 return curr_participant;
             },
             messages: (state) => {
-                let messages = [];
```

Comment:
Those lines can't be removed. They are important to keep the component easy to debug with Vuex Timetravel.
Please, check this issue #6 that explains the problem and this PR #7 that fixes the bug.